worker/Makefile

```diff
@@ -134,7 +134,7 @@ clean-all: clean-subprojects
 
 mediasoup-worker: setup
 ifeq ($(MEDIASOUP_WORKER_BIN),)
-	$(MESON) compile -j $(CORES) -C $(MEDIASOUP_OUT_DIR)/$(MEDIASOUP_BUILDTYPE) mediasoup-worker
+	$(MESON) compile -j $(CORES) $(MEDIASOUP_OUT_DIR)/$(MEDIASOUP_BUILDTYPE) mediasoup-worker
```

Comment:
Why? `-C` specified build directory, you shouldn't remove it.

Reply:
Sorry I'm doing some local debugging -- the build is failing with git but working with a local path dependency. sorry for the noise ill close the PR for now